tests/s390x_tests/lib/common.sh

```diff
@@ -1,999 +0,0 @@
-# Copyright (C) 2018 IBM Corp.
```

Comment:
quick question - why were those files removed?

Reply:
Hi Matthias, we splitted this script to small scripts (https://github.com/elifaln/os-autoinst-distri-opensuse/tree/s390x_tests/tests/s390x_tests/lib).